storio-content-resolver-annotations-processor/src/main/java/com/pushtorefresh/storio/contentresolver/annotations/processor/StorIOContentResolverProcessor.java

```diff
@@ -295,6 +300,12 @@ protected void validateAnnotatedClassesAndColumns(@NotNull final Map<TypeElement
 
     @NotNull
     @Override
+    protected Class<? extends Annotation> getCreatorAnnotationClass() {
+        return null;
```

Comment:
I haven't touched `StorIOContentResolverProccessor` yet and was forced to modify it since I've changed it's super class.